trafilatura/downloads.py

```diff
@@ -90,8 +89,8 @@ def _send_request(url, no_ssl, config):
     global HTTP_POOL, NO_CERT_POOL, RETRY_STRATEGY
     if not RETRY_STRATEGY:
         RETRY_STRATEGY = urllib3.util.Retry(
-            total=0,
-            redirect=MAX_REDIRECTS, # raise_on_redirect=False,
+            total=config.getint("DEFAULT", "MAX_REDIRECTS"),
```

Comment:
I had to allow at least as many retries as the max redirects here.
I wonder if we should set max_redirects to 0 as the new default because it was effectively 0 as it didn't work well

Reply:
It's better to keep the intended logic.